pkg/services/policer/queue.go

```diff
@@ -1,6 +1,8 @@
 package policer
 
 import (
+	"math/rand"
```

Comment:
We have an implementation of math/rand with cryptographic source, let's use it.
Examples: 
https://github.com/nspcc-dev/neofs-node/blob/8eadf18e015981dd8a11e4ce661e6fc2b5388a2e/pkg/services/audit/auditor/por.go#L70-L72
https://github.com/nspcc-dev/neofs-node/blob/0bcc5056828d8cc00bc921953ca5f8a682e22183/pkg/innerring/processors/audit/process.go#L76-L81